### ISSUE TITLE
A new board is empty

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/kiri#10]
A fresh board is empty.

```
    if False:
        # Issue 10
        b = Board()
        assert b.is_empty(), "A new board is empty"
```

 * Activate the test
 * Remove the comment
 * Make the test pass by making `Board::is_empty` return `True` for now
 * Add, commit, push

**Comment by Kirsten29** [comment]
#10 fixed

**Comment by richelbilderbeek** [comment]
@Kirsten29: nice try!

Sadly, the build fails [on GHA](https://github.com/richelbilderbeek/kiri/runs/1634255879?check_suite_focus=true#step:9:14), hence I re-open this Issue.

Good luck on your next attempt :+1:

**Comment by Kirsten29** [comment]
Added those pesky parentheses to fix #10.
